damus/Nostr/RelayConnection.swift

```diff
@@ -114,51 +114,47 @@ class RelayConnection: WebSocketDelegate {
 
         handleEvent(.ws_event(event))
     }
-
 }
 
-func make_nostr_req(_ req: NostrRequest) -> String? {
-    switch req {
-    case .subscribe(let sub):
-        return make_nostr_subscription_req(sub.filters, sub_id: sub.sub_id)
-    case .unsubscribe(let sub_id):
-        return make_nostr_unsubscribe_req(sub_id)
-    case .event(let ev):
-        return make_nostr_push_event(ev: ev)
+protocol NostrRequestMaking {}
+extension NostrRequestMaking {
```

Comment:
I'm actually not a fan of this pattern. This just looks like class extensions. I'm pretty happy with pure top-level functions, don't need to get fancy here. I suggest in the future to make all of these changes separate so I can merge them individually.

Reply:
That's okay. I can back out that part. Wanted to see what you thought of the pattern.

Reply:
Done. I'll leave that out next time.